lib/src/test/java/de/siegmar/fastcsv/util/LimitsTest.java

```diff
@@ -0,0 +1,67 @@
+package de.siegmar.fastcsv.util;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class LimitsTest {
```

Comment:
The two properties (`fastcsv.max.field.size` and `fastcsv.max.field.count`) should be declared as constants in order to not repeat them over and over again...